docs/upgrade.md

```diff
@@ -18,24 +18,28 @@ Before upgrading the pools make sure the following prerequisites are taken care
 
  - Upgrade the control plane components by applying the desired version of cstor-operator from the [charts](https://github.com/openebs/charts/tree/gh-pages). 
```

Comment:
update the cstor-operators link here instead of branch ?

Reply:
No, If we point then it will be specific to a particular version which we don't want to confuse users. Let it me branch so that users can select desired one according to their usage.

Reply:
yeah , make sense.